site/content/en/docs/concepts/workload.md

```diff
@@ -128,6 +128,24 @@ status:
 ```
 The `count` can only increase while the workload holds a Quota Reservation.
 
+## Exponential Backoff ReQueueing for Sequential Admission with Ready Pods
```

Comment:
Before going into backoffs, we should have a section for ReadyPodsTimeout in general.

Reply:
That makes sense.
We didn't add any docs related to PodsReady when we added the waitForPodsReady feature :(

Reply:
not on this page.... most of it is described in tasks. We can keep the note here minimal.

Reply:
alright, I will do it.

Reply:
Done.